controllers/buildpipeline/buildpipeline_adapter.go

```diff
@@ -204,12 +205,12 @@ func (a *Adapter) prepareSnapshotForPipelineRun(pipelineRun *tektonv1beta1.Pipel
 
 	// Copy PipelineRun PAC annotations/labels from Build to snapshot.
 	// Modify the prefix so the PaC controller won't react to PipelineRuns generated from the snapshot.
-	h.CopyLabelsByPrefix(&pipelineRun.ObjectMeta, &snapshot.ObjectMeta, "pipelinesascode.tekton.dev", gitops.PipelinesAsCodePrefix)
-	h.CopyAnnotationsByPrefix(&pipelineRun.ObjectMeta, &snapshot.ObjectMeta, "pipelinesascode.tekton.dev", gitops.PipelinesAsCodePrefix)
+	_ = metadata.CopyLabelsWithPrefixReplacement(&pipelineRun.ObjectMeta, &snapshot.ObjectMeta, "pipelinesascode.tekton.dev", gitops.PipelinesAsCodePrefix)
+	_ = metadata.CopyAnnotationsWithPrefixReplacement(&pipelineRun.ObjectMeta, &snapshot.ObjectMeta, "pipelinesascode.tekton.dev", gitops.PipelinesAsCodePrefix)
```

Comment:
I don't have such function in main branch and I don't see it defined here. Is it planned in different PR?

Reply:
This function is imported from the [operator-toolkit repo](https://github.com/redhat-appstudio/operator-toolkit/tree/main/metadata), that's why it's not visible in our code.